modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -36,6 +36,11 @@ set(TFM_VALID_PARTITIONS
 #                TF-M regression tests
 # BL2: Boolean if the TF-M build uses MCUboot. Default: True
 # ENABLED_PARTITIONS: List of TFM partitions to enable.
+# CMAKE_ARGS: Additional CMake flags to be used when building TF-M
```

Comment:
@tejlmand @erwango would this be better to be called CMAKE_ADDITIONAL_ARGS, or CMAKE_CUSTOM_ARGS, to reflect better what it is used for?

Reply:
well, this is the description of the function: `trusted_firmware_build()`

So from the point of view of that function, it's just CMake args.

You could ideally use that function from other locations.

But when called from the internal Zephyr build here: https://github.com/ioannisg/zephyr/blob/b6678723a282a99e62aebbc62570e37748c03df9/modules/trusted-firmware-m/CMakeLists.txt#L321
```
    CMAKE_ARGS $<GENEX_EVAL:$<TARGET_PROPERTY:zephyr_property_target,TFM_CMAKE_OPTIONS>>
```
you may consider it custom args.